pkg/queue/request_metric.go

```diff
@@ -206,7 +208,7 @@ func (h *appRequestMetricsHandler) ServeHTTP(w http.ResponseWriter, r *http.Requ
 }
 
 // GetRouteTagNameFromRequest extracts the value of the tag header from http.Request
-func GetRouteTagNameFromRequest(r *http.Request) string {
+/*func GetRouteTagNameFromRequest(r *http.Request) string {
```

Comment:
Are you commenting these simply due to coverage limitations?

Reply:
Yes.  It's was below the threshold.